src/components/FilterTabs/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC } from "react"
+import { Button, ButtonGroup, useColorModeValue } from "@chakra-ui/react"
+
+export type Variant = "primary" | "outline"
+export type Size = "xs" | "sm" | "md" | "lg"
+
+export interface FilterTabProps {
+  title: string
+  tabId: string
+  onClick: (tabId: string) => void
+  selectedTabId: string
+  variant: Variant
+  size: Size
+}
+
+export interface FilterTabsProps {
+  variant?: Variant
+  size?: Size
+  selectedTabId: string
+  setSelectedTabId: (tabId: string) => void
+  tabs: {
+    title: string
+    tabId: string
+    onClick?: (tabId: string) => void
+  }[]
+}
+
+const FilterTab: FC<FilterTabProps> = ({
+  selectedTabId,
+  tabId,
+  onClick,
+  title,
+  variant,
+  size,
+}) => {
+  const isActive = selectedTabId === tabId
+
+  const activeColor = useColorModeValue(
+    variant === "primary" ? "brand.500" : "gray.700",
+    variant === "primary" ? "brand.100" : "gray.50"
+  )
+  const passiveColor = "gray.500"
+
+  const activeBg = useColorModeValue(
+    variant === "primary" ? "brand.50" : "white",
+    variant === "primary" ? "brand.800" : "gray.900"
+  )
+
+  const activeBorderColor = useColorModeValue("gray.50", "gray.300")
+
+  const passiveHoverBg = useColorModeValue(
+    variant === "primary" ? "brand.50" : "white",
+    variant === "primary" ? "brand.800" : "gray.800"
+  )
```

Comment:
I think we should build this component in a chakra way and thus create a theme for it and put all those styles there. 

I think (although I'm not sure) that if we pass the props to the `baseStyles` in theme we can then use `prop.isActive` there and set the proper colours in the theme itself.

I hope you know what i mean. Also check themes that we already have in the lib like `CheckListItem` icon or `IconSpinner` theme.

Also @r-czajkowski I'm curious what you think about this idea.

Reply:
this is a good suggestion. It's very easy to pass props via `useStyleConfig` and it separates our styling logic from the component. See: https://github.com/threshold-network/components/pull/7/commits/84482673b837e4f7e7227cd7ec01708921d17845